src/v2/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -169,10 +195,11 @@ export interface TabCounts {
   "Artist Series"?: number
 }
 
-export const tabCountMap: (props: Props) => TabCounts = props => {
+export const tabCountMap: (
+  searchableConnection: NavigationTabs_searchableConnection
+) => TabCounts = props => {
   return Object.entries(TAB_NAME_MAP).reduce((acc, [key, val]) => {
-    // @ts-expect-error STRICT_NULL_CHECK
-    let count = get(aggregationFor(props, key), agg => agg.count, 0)
+    let count = get(aggregationFor(props, key), agg => agg?.count, 0)
```

Comment:
`non-blocking:` if you see these `get` calls you can just replace them with the safe operator

```suggestion
    let count = aggregationFor(props, key)?.count ?? 0
```